### PR TITLE
5.05c, 5.06, 5.07a Add Actions section to submission detail page

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

File: src/components/submission/audit/list.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        this.xmlFormId,
+        this.instanceId
+      );
+    },
+    auditsToShow() {
+      if (this.audits == null) return null;
+      return this.audits.filter(audit =>
+        (audit.action === 'submission.update' && audit.details.reviewState != null) ||
+        audit.action === 'submission.update.version');
+    }
```
Comment (matthew-white) [inline src/components/submission/audit/list.vue:88]: Gray cloud icon with text that says "Submitted"

Comment (matthew-white) [reply src/components/submission/audit/list.vue:88]: Also received icon.

File: src/components/submission/update-review-state.vue [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    instanceId: {
+      type: String,
+      required: true
+    }
+  },
+  data() {
+    const { reviewState } = this.$store.state.request.data.submission.__system;
+    return {
+      awaitingResponse: false,
+      reviewState: reviewState != null ? reviewState : 'hasIssues',
```
Comment (matthew-white) [inline src/components/submission/update-review-state.vue:96]: hasIssues -> approved

File: src/components/submission/audit/list.vue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        </template>
+      </template>
+    </template>
+    <template #body>
+      <submission-audit-table :audits="auditsToShow"/>
+      <loading :state="$store.getters.initiallyLoading(['audits'])"/>
+      <p v-if="audits != null && auditsToShow.length === 0"
+        class="empty-table-message">
+        {{ $t('emptyTable') }}
+      </p>
```
Comment (matthew-white) [inline src/components/submission/audit/list.vue:42]: Remove.

**Review by issa-tseng — APPROVED** [review]

File: test/components/submission/update-review-state.spec.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    requestData: { submission },
+    ...options
+  };
+};
+const mountComponent = (options = undefined) =>
+  mount(SubmissionUpdateReviewState, mountOptions(options));
+const mockHttpForComponent = (options = undefined) =>
+  mockHttp().mount(SubmissionUpdateReviewState, mountOptions(options));
+
+describe('SubmissionUpdateReviewState', () => {
```
Comment (matthew-white) [inline test/components/submission/update-review-state.spec.js:29]: Create test data `submission.create` audit log entries.

**Review by matthew-white — COMMENTED** [review]
Notes from interactive review

**Pull request merged**